baseplate/sidecars/trace_publisher.py

```diff
@@ -87,29 +110,34 @@ def publish(self, payload: SerializedBatch) -> None:
         for _ in RetryPolicy.new(attempts=self.retry_limit):
             try:
                 with self.metrics.timer("post"):
-                    response = self.session.post(
-                        self.endpoint,
-                        data=payload.serialized,
-                        headers=headers,
-                        timeout=self.post_timeout,
-                        stream=False,
-                    )
+                    with publishLatency.time():
+                        response = self.session.post(
+                            self.endpoint,
+                            data=payload.serialized,
+                            headers=headers,
+                            timeout=self.post_timeout,
+                            stream=False,
+                        )
                 response.raise_for_status()
             except requests.HTTPError as exc:
+                publishErrors.labels(error_type="http").inc()
                 self.metrics.counter("error.http").increment()
                 response = getattr(exc, "response", None)
                 if response is not None:
                     logger.exception("HTTP Request failed. Error: %s", response.text)
                     # If client error, skip retries
                     if response.status_code < 500:
+                        publishErrors.labels(error_type="http.client").inc()
```

Comment:
I see this matches the statsd error, but what is an "http.client" error? That's not really descriptive enough for us to take action on.